media/js/externalpages/pocket/mobile-nav.es6.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+const navOpenBtn = document.querySelector('.global-nav-mobile-menu-btn');
+const mobileNavWrapper = document.querySelector('.mobile-nav-wrapper');
+const mobileNav = document.querySelector('.mobile-nav');
+const navCloseBtn = document.querySelector('.mobile-nav-close-btn');
+
+const TAB = 9;
+const ESC = 27;
+
+function noScroll() {
+    window.scrollTo(0, 0);
```

Comment:
This seems to work OK, but the background is kinda glitchy when I try to scroll on iOS. Not a blocker though.

Reply:
Nice catch, made a change to the scroll which should solve both issues